.github/workflows/integration_test.yaml

```diff
@@ -51,8 +51,24 @@ on:
         default: statefulset
       chaos-duration:
         type: string
-        description: Duration of the chaos experiment
+        description: |
+          Duration of the chaos experiment (added to 600s for go test timeout)
         default: 60
+      chaos-interval:
+        type: string
+        description: Interval to attempt re-run of an experiment
+        default: 30
+      chaos-status-duration:
+        type: string
+        description: |
+          Duration used for status check retries
+          Retry is chaos-duration/chaos-delay times and each retry
+          sleeps chaos-delay
+        default: 90
+      chaos-status-delay:
```

Comment:
Would you mind ordering the inputs alphabetically?

Reply:
Good point and adds to readability, I'll order them - thanks!